pkg/deviceshifu/deviceshifusocket/deviceshifusocketconfig.go

```diff
@@ -11,3 +11,12 @@ type ReturnBody struct {
 	Message string `json:"message"`
 	Status  int    `json:"status"`
 }
+
+const (
+	EncodeUTF8Str = "utf-8"
+	EncodeHexStr  = "hex"
```

Comment:
let's put encoding/decoding support inside `EdgeDevice` definition since it belongs to CRD configuration